### PR TITLE
fix: optimize vote processing

### PR DESCRIPTION
**Pull request by mfrankovi**
## Purpose

Optimized RLP votes processing.
Removed verify method in VrfPbftSortition which actually did not do anything since there was no check for return value.

These changes significantly reduce time of processing vote packets

## Version Notes ##
{ major | minor | patch }

**NOTE: Include the version notes in the squash commit message**

### CODE REVIEW

**Review by gaoqi8649 — COMMENTED** [review]
I just checked the code. I think CCL put the verify function there that is not for verification, that is for setup “output” and “threshold” value

**Review by gaoqi8649 — APPROVED** [review]
In vote verification has the verify() to setup output value. Should remove here

File: src/consensus/vote.cpp [hunk 1]
```diff
@@ -311,7 +314,7 @@ std::vector<Vote> VoteManager::getVerifiedVotes(uint64_t const pbft_round, size_
     }
   }
 
-  for (auto const& v : votes_to_verify) {
+  for (auto& v : votes_to_verify) {
```
Comment (JakubFornadel) [inline src/consensus/vote.cpp:317]: why did you remove const ?

File: src/consensus/vote.cpp [hunk 2]
```diff
@@ -485,7 +487,7 @@ bool VoteManager::pbftBlockHasEnoughValidCertVotes(PbftBlockCert const& pbft_blo
   std::vector<Vote> valid_votes;
   auto first_cert_vote_round = pbft_block_and_votes.cert_votes[0].getRound();
 
-  for (auto const& v : pbft_block_and_votes.cert_votes) {
+  for (auto& v : pbft_block_and_votes.cert_votes) {
```
Comment (JakubFornadel) [inline src/consensus/vote.cpp:490]: here too - remove const ?

**Review by MatusKysel — APPROVED** [review]
